externals/kyuubi-chat-engine/src/main/scala/org/apache/kyuubi/engine/chat/session/ChatSessionManager.scala

```diff
@@ -59,12 +59,12 @@ class ChatSessionManager(name: String)
   override def closeSession(sessionHandle: SessionHandle): Unit = {
     super.closeSession(sessionHandle)
     if (conf.get(ENGINE_SHARE_LEVEL) == ShareLevel.CONNECTION.toString) {
-      info("Session stopped due to shared level is Connection.")
-      stopSession()
+      info("Chat engine stopped due to session stopped and shared level is Connection.")
```

Comment:
please UPPER CASE the CONNECTION

```suggestion
      info("Chat engine stopped due to session stopped and shared level is CONNECTION.")
```